### DISCUSSION TITLE
Openstack swift large object overwrite deletes ".file-segments" folder

### DISCUSSION THREAD

**Discussion by OdedRaiches** [iterate-ch/cyberduck#14311] [Q&A] [answered]
Hi, I'm new into Cyberduck, was mostly using s3 tools to integrate with my object storage point.
Have a customer that uses this client to integrate with the swift protocol (keystone 3).

Noticed that when uploading a large object there is a   `.file-segments`  folder for holding the segments of the object.
But, when overwriting the same object, there is a  `DELETE bulk-delete=1`  coming in after uploading the segments again and it removes the  `.file-segments`  folder, which makes the object not usable (only the manifest object stays intact, and all the actual data is gone).

Found in Cyberduck version 8.5.6 (39394).

Thanks for the support!

**Comment by dkocher** [comment, marked as answer]
As a workaround for this bug set the [preference](https://docs.cyberduck.io/cyberduck/preferences/#hidden-configuration-options) `openstack.upload.largeobject.cleanup` to `false`.

**Reply by OdedRaiches** [reply]
thanks!
Is there any chance this will be dealt with in the next release? didn't find any bug report about it

**Reply by dkocher** [reply]
I have opened #14326.
